### PR TITLE
Increase timeout for some flaky tests

### PR DESCRIPTION
**Pull request by choumx**
Also clean up test-amp-list.js and add test for both versions of amp-mustache.

### CODE REVIEW

File: test/integration/test-amp-analytics.js [hunk 1]
```diff
@@ -17,9 +17,8 @@
 import {BrowserController, RequestBank} from '../../testing/test-helper';
 import {parseQueryString} from '../../src/url';
 
-describe.configure().run('amp' +
-    '-analytics', function() {
-  this.timeout(5000);
+describe.configure().run('amp-analytics', function() {
+  this.timeout(10000);
```
Comment (choumx) [inline test/integration/test-amp-analytics.js]: This used to be 15000.

**Comment by lannka** [comment]
for describes.integration tests, does it make sense to have the default timeout 15s on Travis and 5s locally?

File: extensions/amp-list/0.1/test/integration/test-amp-list.js [hunk 2]
```diff
@@ -17,54 +17,58 @@
...
 
-const body =
-    '<amp-list id="list" width=300 height=100 ' +
-        'src="http://localhost:9876/list/fruit-data/get?cors=0">' +
-      '<template type="amp-mustache">' +
-        '{{name}} : {{quantity}} @ ${{unitPrice}}' +
-      '</template>' +
-    '</amp-list>';
-
+const TIMEOUT = 12500;
```
Comment (rsimha) [inline extensions/amp-list/0.1/test/integration/test-amp-list.js]: This appears to be insufficient on Chrome 71. Could it be that something else is going wrong?

Comment (choumx) [reply extensions/amp-list/0.1/test/integration/test-amp-list.js]: This was actually a bug. 🙈

**Pull request merged**